src/pygeosx/unitTests/testPygeosxAcoustic.py

```diff
@@ -0,0 +1,32 @@
+import pygeosx
+from mpi4py import MPI
+import numpy as np
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+def test_forwardPropagationAcoustic():
+    argv = ["testPygeosxAcoustic.py", "-i", "acous_5x5x5.xml"]
+    solver = "acousticSolver"
+    forwardPropagation(argv, solver)
+
+def forwardPropagation(argv, solver):
```

Comment:
Could you add docstrings to any functions you are adding to python files?  The format is up to you, but I personally prefer the google style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html